build.gradle

```diff
@@ -215,7 +215,7 @@ def versions = [
   camel               : '3.8.0',
   gradlePitest        : '1.5.1',
   pitest              : '1.7.0',
-  log4j               : '2.15.0'
+  log4j               : '2.16.0'
```

Comment:
I think this is already `2.16.0` in master

Reply:
merged master with current branch